CONTRIBUTING.md

```diff
@@ -4,8 +4,33 @@
 1. All python code must be python 2.6/2.7/3.6 compatible
 1. The code should follow linting from pylint
 
-## Unit tests
-Our unit tests are run within containers. To first create the container images, run:
+## Developing locally
+
+The command bellow creates a virtual environment and installs needed dependencies.
+```bash
+# for CentOS8-like system
+make install  # for CentOS 8-like system
+# for CentOS7-like system
+# CENTOS_VER=7 make install
+# for CentOS6-like system
+#. .venv2.7/bin/activate
+# CENTOS_VER=6 make install
+#. .venv2.6/bin/activate
+```
```

Comment:
```suggestion
The commands bellow activate a virtual environment with all the necessary dependencies installed:
```bash
# for CentOS8-like system
make install  # for CentOS 8-like system
. .venv3/bin/activate
# or for CentOS7-like system
CENTOS_VER=7 make install
. .venv2.7/bin/activate
# or for CentOS6-like system
CENTOS_VER=6 make install
. .venv2.6/bin/activate
```

Reply:
Now this chapter looks simplier:
```md
## Developing locally

The commands below installs a python3 virtual environment
with all the necessary dependencies installed:
```bash
make install
```

and you're ready to run tests with:
```bash
make tests-locally
```
```